lib/main.dart

```diff
@@ -30,11 +38,21 @@ class _AppScaffoldState extends State<AppScaffold> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(title: const Text('Solaris')),
+      appBar: AppBar(
+        title: const Text('Solaris'),
+        actions: [
+          IconButton(
+            icon: const Icon(Icons.logout),
+            onPressed: () {
+              context.read<LoginCubit>().logout();
+            },
+          )
+        ],
+      ),
       body: widget.child,
       bottomNavigationBar: BottomNavigationBar(
-        currentIndex: calculateSelectedIndex(context),
-        onTap: (value) => onTap(value, context),
+        currentIndex: AppRouter.calculateSelectedIndex(context),
+        onTap: (value) => AppRouter.onTap(value, context),
```

Comment:
`AppRouter.onTap` is generic (is not easy to figure out what it does unless you check the implementation). Can we chose a more subjective / intuitive name?

The final purpose would be to easily figure out what would happen on `onTap` function from the `BottomNavigationBar` without checking all the code.

Here's an example:
```
      bottomNavigationBar: BottomNavigationBar(
        ....
        onTap: (pageIndex) => navigateToPage(pageIndex, context),
```
These is just an example. You can pick other names that seem suggestive to you.

Reply:
@ilie-lupu-ts @szaszgyozo

Reply:
Fixed in [this](https://github.com/ivoryio/SolarisDemoApp/pull/3) PR.